Dockerfile

```diff
@@ -13,7 +13,8 @@ RUN apk --no-cache add --virtual build-deps \
   linux-headers \
   xz-libs \
   tzdata \
-  yarn
+  yarn \
+  gcompat
 
 # Alpine does not have a glibc, and this is needed for dart-sass
```

Comment:
Maybe this can now be also removed assuming `gcompat` is adding the glibc compatibility?

I can try to test in a fresh M1 (my personal laptop).

Reply:
In my non-M1 machine, it keeps working removing all these lines.
Maybe quickest for you to test in your M1 @timpeat ?

Reply:
Perfect! Will test and update!

Reply:
M1 still requires gcombat sadly. You are correct on the dart-sass fix though.